projects/ngrx.io/content/guide/migration/v8.md

```diff
@@ -18,16 +18,36 @@ V8 has the minimum version requirements:
 - TypeScript version 3.3.x
 - RxJS version 6.x
 
-## @ngrx/store
+## Highlighted new features
 
-### `META_REDUCERS` token
+- [@ngrx/data](): the [angular-ngrx-data](https://github.com/johnpapa/angular-ngrx-data) library by John Papa and Ward Bell is now an official part of NgRx and can be installed with `npm install @ngrx/data`
```

Comment:
Once #1767 is merged, we can point to the docs. (build is failing at the moment because the url does not exist yet)

Reply:
Great idea, we go a little deeper in the blog post, so it's probably better to just point to it 👍